### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by dober2** [rmrector/script.artwork.beef#39]
ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.ValueError'>
                                            Error Contents: invalid literal for int() with base 10: ''
                                            Traceback (most recent call last):
                                              File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.artwork.beef\service.py", line 393, in <module>
                                                ArtworkService().run()
                                              File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.artwork.beef\service.py", line 104, in run
                                                successful = self.process_allvideos(self.processed.is_stale)
                                              File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.artwork.beef\service.py", line 254, in process_allvideos
                                                return self.processor.process_medialist(items)
                                              File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.artwork.beef\lib\artworkprocessor.py", line 232, in process_medialist
                                                return self.process_chunkedlist([medialist], 1, alwaysnotify)
                                              File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.artwork.beef\lib\artworkprocessor.py", line 247, in process_chunkedlist
                                                self._process_chunk(medialist, len(medialist) == 1 and not idx, alwaysnotify)
                                              File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.artwork.beef\lib\artworkprocessor.py", line 270, in _process_chunk
                                                services_hit = self._process_item(mediaitem, singleitem)
                                              File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.artwork.beef\lib\artworkprocessor.py", line 309, in _process_item
                                                services_hit, error = self.gatherer.getartwork(mediaitem, onlyfs, auto)
                                              File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.artwork.beef\lib\gatherer.py", line 27, in getartwork
                                                mediaitem.uniqueids, mediaitem.missingart)
                                              File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.artwork.beef\lib\gatherer.py", line 94, in get_external_artwork
                                                if int(season) not in seasons:
                                            ValueError: invalid literal for int() with base 10: ''
                                            -->End of Python script error report<--

**Comment by str8upx** [comment]
Same error here. Artwork beef refused to open manage/select artwork saying error (log showed same as above). I ended up deleting kodi and starting over with 18.3 (android) and Beef wouldn't install. Whipped again and got it installed but now no matter what artwork I try to add/change it just says "no artwork found". I've entered my API keys and no change. On my win 10 htpc kodi 18.2 Beef works without issue. Not sure if it's related to the Nvidia Shield Pie update a few days ago because I had not tried to edit artwork for awhile. I've tried changing every setting imaginable and just cant make it work.
